ytypes/leaf_list.go

```diff
@@ -119,6 +124,8 @@ func unmarshalLeafList(schema *yang.Entry, parent interface{}, value interface{}
 		if len(sa.LeaflistVal.GetElement()) == 0 {
 			return fmt.Errorf("unmarshalLeafList for schema %s: value %v: got empty leaf list, expect non-empty leaf list", schema.Name, util.ValueStr(value))
 		}
+		// A new leaf-list update specifies the entire leaf-list
```

Comment:
Suggest something like `// We are supplied the entire leaf-list, so we should clear its contents if it is non-nil`.

Reply:
thanks.